server/src/test/java/org/apache/druid/test/utils/ImmutableDruidDataSourceTestUtils.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.test.utils;
+
+import org.apache.druid.client.ImmutableDruidDataSource;
+import org.junit.Assert;
+
+import java.util.List;
+
+public class ImmutableDruidDataSourceTestUtils
+{
+
+  /**
+   * This method is to check equality of {@link ImmutableDruidDataSource} objects to be called from test code.
+   * @param expected expected object
+   * @param actual actual object
+   *
+   */
+  public static void assertEqualsImmutableDruidDataSource(ImmutableDruidDataSource expected, ImmutableDruidDataSource
+    actual)
+  {
+    if (checkEquals(expected, actual)) {
+      return;
+    } else {
+      throw new AssertionError("Expected and actual objects are not equal as per ImmutableDruidDataSource's " +
+        "equalsForTesting() method");
+    }
+  }
+
+  private static boolean checkEquals(ImmutableDruidDataSource expected, ImmutableDruidDataSource actual)
+  {
+    if (expected == null) {
+      return actual == null;
+    }
+
+    return expected.equalsForTesting(actual);
+  }
+
+  /**
+   * This method is to check the equality of a list of {@link ImmutableDruidDataSource} objects to be called from
+   * test code
+   * @param expected expected list
+   * @param actual actual list
+   * @return
+   */
+  public static boolean assertEqualsImmutableDruidDataSource(List<ImmutableDruidDataSource> expected,
```

Comment:
In Druid style, method signature doesn't fit a single line all params should be chopped:
```suggestion
  public static boolean assertEqualsImmutableDruidDataSource(
      List<ImmutableDruidDataSource> expected,
      List<ImmutableDruidDataSource> actual
  )
  {
```

Reply:
Chopped off where all params do not fit in a single line in other places as well.

Reply:
Thanks.